demonstrations/tutorial_qft_arithmetics.py

```diff
@@ -13,18 +13,23 @@
 
 
 
-*Author: Guillermo Alonso-Linaje. Posted:  2022. Last updated: 13 June 2022*
+*Author: Guillermo Alonso-Linaje.*
 
-Arithmetic is a fundamental branch of mathematics that consists of the study of the main operations on numbers such
-as addition, multiplication, subtraction or division. Using arithmetic operations we can understand the world around us and
-solve many of our daily tasks.
+*Posted:  2022. Last updated: 13 June 2022*
 
-In computing, arithmetic is crucial to the implementation of any type of algorithm. Therefore, we will
-make an approach to arithmetic in quantum computers using the quantum Fourier transform (QFT).
-In this demo, we will not focus on understanding how the QFT is built,
+Arithmetic is a fundamental branch of mathematics that consists of the study of the main operations with numbers such as
+addition, multiplication, subtraction and division. Through arithmetic operations we can understand the world around us
+and solve many of our daily tasks.
+
+In computer science, arithmetic is crucial for the implementation of any kind of algorithm and in quantum computing
+this is also the case. For this reason, throughout the demo we are going to do
+an approach to arithmetic in quantum computers.  The simplest and most direct way to achieve this goal is to use the
+quantum Fourier transform (QFT), so we will give the basic ideas of how to do it.
+
+In this demo, we will not focus on understanding how the QFT is built
 as we can find a great explanation in the
 `Codebook <https://codebook.xanadu.ai/F.1>`__. Instead, we will develop the
-intuition for how it works and how we can take advantage of it!
+intuition of how it works and how we can take advantage of it.

```

Comment:
```suggestion
intuition for how it works and how we can best take advantage of it.
```
Alternatively you could use 'about'.